src/System.Management.Automation/resources/ParserStrings.resx

```diff
@@ -493,6 +493,9 @@ The correct form is: foreach ($a in $b) {...}</value>
   <data name="DuplicateScriptCommandClause" xml:space="preserve">
     <value>Script command clause '{0}' has already been defined.</value>
   </data>
+  <data name="MissingEndCurlyBraceOrNamedBlock" xml:space="preserve">
+    <value>Missing closing '}' in script block. If you meant to continue with the script block, please note that new statements should be put in named blocks after a named block has been declared.</value>
```

Comment:
Sorry, it's hard to understand 😕

Reply:
How about `Missing closing '}' in script block. If you meant to add more statements to the script block, put the statements in one or more named blocks.`